metaflow/environment.py

```diff
@@ -77,20 +77,41 @@ def get_client_info(cls, flow_name, metadata):
         """
         return "Local environment"
 
+    def get_boto3_copy_command(self, s3_path, local_path, command="download_file"):
```

Comment:
Thanks for the PR @talebzeghmi! Looks like we don't have a use case for `upload_file` at the moment. Can we remove that?